server/rollback.go

```diff
@@ -30,14 +30,14 @@ application.
 			if err != nil {
 				return err
 			}
+			app := appCreator(ctx.Logger, db, nil, ctx.Viper)
```

Comment:
need this to initialize the multistore properly.

Reply:
and it needs to be implemented by the third-party app, it'll be a big breaking API change I think.